style.css

```
@@ -1,8 +1,169 @@
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&family=Rubik:wght@600&display=swap');
+
+
+:root {
+  --primary-color: #026a90;
+  --secondary-color: lightgray;
+}
+
+* {
+  box-sizing: border-box;
+
+}
+
+body {
+  background-color: #FFFFFF;
+  font-family: sans-serif;
+  margin: 0;
+}
+
+header{
+  margin: 0 auto;
+  width: 100%;
   text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
 }
+h1{
+  margin: 0 0 50px;
+  font-size: 48px;
+  font-weight: 400;
+
+
+font-family: 'Roboto Mono', monospace;
+  color: #000;
+}
+
+.search {
+  background-image: url('search.png');
+  background-size: 20px;
+  background-position: calc(100% - 10px) center;
+  background-repeat: no-repeat;
+  background-color: white;
+  border: 1px solid black;
+  border-radius: 0px;
+  font-size: 1rem;
+  padding: 0.5rem;
+  color: black;
+}
+
+
+.search::placeholder {
+  color: black;
+}
+.search:focus::placeholder {
+  color: transparent;
+}
+
+.search:focus {
+  outline: none;
+}
+
+
+#main{
+  display: flex;
+  justify-content: center;
+  max-width: 1200px;
+  width:100%;
+  margin: 80px auto 0;
+  flex-wrap: wrap;
+  gap:48px 75px;
+}
+
+.card {
+  display: block;
+  cursor: pointer;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+position: relative;
+
+height: 374px;
+width: 350px;
+box-shadow: 0px 10px 20px 5px rgba(0, 0, 0, 0.25);
+border-radius: 20px;
+}
+
+
+
+.title{
+  position: absolute;
+  left: 50%;
+  
+  top: 50%;
+  transform: translate(-50%,-50%);
+  
+  font-family: 'Rubik';
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 28px;
+  letter-spacing: 0.004em;
+  color: #FFFFFF;
+}
+
+
+
+.button{
+  position: absolute !important;
+  left:50%;
+  transform: translateX(-50%);
+  
+  bottom: 56px;
+  font-family: 'Rubik';
```

```suggestion
  font-family: 'Rubik', sans-serif;
```

Please add sans-serif along with 'Rubik" font to work as a fallback.